control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -2554,9 +2562,18 @@ func (r *HostedControlPlaneReconciler) reconcileOpenShiftAPIServer(ctx context.C
 		r.Log.Info("reconciled openshift apiserver servicemonitor", "result", result)
 	}
 
+	serviceServingCA := manifests.ServiceServingCA(hcp.Namespace)
```

Comment:
It's not related, but it's part of the refactoring. Instead of the separate CMCA controller update the apiserver deployment, the configmap is read here, part of the reconcile and the only thing that changes the deployment is the reconcile function.